.github/workflows/nightly-upstream-snapshot-build.yml

```diff
@@ -98,6 +98,9 @@ jobs:
           name: aws-opentelemetry-agent.jar
           path: otelagent/build/libs/aws-opentelemetry-agent-*.jar
 
+    concurrency:
+      group: e2e-adot-test
+      cancel-in-progress: false
```

Comment:
The E2E test call is missing here, so the concurrency tag that is meant for it is going into the wrong part of the workflow. Compare with `main` to see the desired e2e test workflow call, not sure which commits are needed for that.